oci/spec_opts.go

```diff
@@ -894,9 +894,9 @@ func WithAppendAdditionalGroups(groups ...string) SpecOpts {
 			if err != nil {
 				return err
 			}
-			ugroups, err := user.ParseGroupFile(gpath)
-			if err != nil {
-				return err
+			ugroups, groupErr := user.ParseGroupFile(gpath)
+			if groupErr != nil && !os.IsNotExist(groupErr) {
+				return groupErr
```

Comment:
Thinking a bit more about this, I think we want to ignore this error completely, no matter what the error is unless the file is needed. WDYT?

Reply:
imho it would be good to keep this error check and return early in case of parsing error. looking into `user.ParseGroupFile` impl it does not seem to guarantee nil result on error. so the remaining logic would potentially be processing partially parsed results.